content/blog/2019/root-transition/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: User guide for extenting Istio root certificate lifetime
+description: The user guide to extend Istio root certificate lifetime.
+publishdate: 2019-06-06
+subtitle:
+attribution: Oliver Liu
+twitter:
+keywords: [security, PKI, certificate, Citadel]
+
+---
+
+## Background
+
+For Istio releases prior to 1.0.8 and prior to 1.1.8, the Istio self-signed certificates have the default lifetime of 1 year.
+If you are using the Istio self-signed certificates with default lifetime configuration,
```

Comment:
`with the default`

Reply:
There is no conditional here when it comes to default lifetime configuration.  We've already established in the line above that this applies to everyone on< 1.1.8, so it just needs to read "If you are using Istio self-signed root certificates,"

Reply:
Removed the default lifetime.